index.html

```diff
@@ -17,24 +17,17 @@
                 </div>
 
                 <p>
-                    <a class="btn btn-primary btn-lg btn-jumbo" href="/documentation/getstarted">Get Started with Flyway
-                        {{site.flywayVersion}} <i class="fa fa-arrow-right"></i></a><br/>
-                </p>
-
-                <p id="links">
-                    <a href="/download">Download</a> -
-                    <a href="/documentation/learnmore/releaseNotes">Release Notes</a> -
-                    <a href="https://github.com/flyway/flyway/blob/master/LICENSE">Apache v2 license</a>
+                    <a class="btn btn-primary btn-lg btn-jumbo" href="/documentation/getstarted">Get Started with Flyway <i class="fa fa-arrow-right"></i></a><br/>
                 </p>
 
                 <div id="worksOn">
                     Works on:
-                    <a href="/documentation/usage/commandline"><i class="fa fa-windows" title="Windows"></i></a>
-                    <a href="/documentation/usage/commandline"><i class="fa fa-apple" title="macOS"></i></a>
-                    <a href="/documentation/usage/commandline"><i class="fa fa-linux" title="Linux"></i></a>
-                    <a href="/documentation/usage/commandline"><img title="Docker" src="/assets/logos/docker.png"></a>
-                    <a href="/documentation/usage/api"><i class="fa fa-coffee" title="Java"></i></a>
-                    <a href="/documentation/usage/api"><i class="fa fa-android" title="Android"></i></a>
+                    <a href="/documentation/usage/commandline"><i class="fa fa-windows" style="font-size: 2em" title="Windows"></i></a>
```

Comment:
Is the style here for changing the size of icons? If so, fontawesome has built in classes that control the icon size: https://fontawesome.com/how-to-use/on-the-web/styling/sizing-icons

Reply:
These dont actually seem to work sadly

Reply:
😢